pom.xml

```diff
@@ -38,12 +38,24 @@
       <artifactId>junit</artifactId>
       <version>4.12</version>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.hamcrest</groupId>
```

Comment:
Sorry, I'm going to be picky here :)

Let's use Truth instead of Hamcrest (https://truth.dev/) (I forgot about Hamcrest or I would have been more clear earlier)

Reply:
Ok, switched to using Truth instead. I used the containsExactlyEntriesIn method with assertThat.